plugins/org.eclipse.osee.jdbc/src/org/eclipse/osee/jdbc/SqlTable.java

```diff
@@ -203,4 +202,25 @@ private StringBuilder getInsertIntoSqlStart() {
       strB.append(") VALUES (");
       return strB;
    }
+
```

Comment:
Can you add some javadoc explaining what this does?

Reply:
Done.